site/docs/v1/tech/core-concepts/integration-points.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: doc
+title: Integration Points
+description: FusionAuth easily integrates with existing or new applications 
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+You typically integrate FusionAuth into one or more applications. User data will live in FusionAuth (possibly synced with other data stores) and users will auth against it. 
+
+FusionAuth is a developer first platform and there are a large number of ways to integrate it into your new or existing applications.
+
+=== Login Options
+
+There are three main ways to have your users sign in: OAuth/OIDC, the Login API, or federated authentication.
+
+==== OAuth/OIDC 
+
+The first option is to use OAuth/OIDC. These are both standards and FusionAuth should work with any library or application which supports them. If you find a library which supports OAuth or OIDC and does not work with FusionAuth, please https://github.com/FusionAuth/fusionauth-issues/issues/[open a bug], as we want to know about it. You can also use a link:/docs/v1/tech/client-libraries/[FusionAuth client library].
+
+Using OAuth/OIDC lets your users authenticate and authorize; they'll receive the responses link:/docs/v1/tech/oauth/[documented for each grant]. 
+
+When you use this option, the data your client receives about the user is limited. You can put custom claims in your JWT using link:/docs/v1/tech/lambdas/[lambdas] if what you need is in the user or the registration objects. 
+
+If this level of integration meets your needs, you'll have more portability and less lock-in to FusionAuth.
+
+==== Login API
+
+You can also use the link:/docs/v1/tech/apis/login[Login API] to sign your users in directly. In this case, you will likely use one of the link:/docs/v1/tech/client-libraries/[FusionAuth client libraries].
+
+When a user signs in with this API, you receive the entire user object, as well as the JWT.
+
+When you build on top of the Login API, you're responsible for building the user interface for all login use cases. This gives you more control, as well. You can still use other FusionAuth provided user interfaces for use cases such as the forgot password flow.
```

Comment:
Seems that we should also point out the down side of using this route which is you now handle credentials which is contrary to the ideal delegation pattern that OIDC provides.